frontend/components/generic/SimpleTable.tsx

```diff
@@ -21,23 +21,28 @@ type TableColumn<D, K> = K extends keyof D
     : TableColumnWithDefinedTransform<D, K>
   : never;
 
+type Comparable = number | string;
```

Comment:
Not sure if the current way we handle nulls is the best. Can we include `null` as a valid `Comparable`?

I think we can always place the nulls at the end, no matter 'asc' or 'desc', what do you think?

Reply:
Agreed, but I've given the caller the option of telling where null should be (start, end, or a sorting based on a NonNullableComparable). By default nulls will be at the bottom, as you suggested.